src/java.base/share/classes/com/sun/crypto/provider/CipherCore.java

```diff
@@ -813,17 +827,8 @@ int doFinal(byte[] input, int inputOffset, int inputLen, byte[] output,
                 int outputOffset)
         throws IllegalBlockSizeException, ShortBufferException,
                BadPaddingException {
-
```

Comment:
Please don't make unnecessary changes relative to upstream. Please restore this blank line and split line 831 back in two.

Reply:
Restored to original.